pkgs/development/libraries/libinih/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "libinih";
+  version = "52";
+
+  src = fetchFromGitHub {
+    owner = "benhoyt";
+    repo = "inih";
+    rev = "r${version}";
+    sha256 = "0lsvm34zabvi1xlximybzvgc58zb90mm3b9babwxlqs05jy871m4";
+  };
+
+  mesonFlags = [
```

Comment:
Why did you remove `"-Dwith_INIReader=true"`?

Reply:
It only occured to me to search the tracker for xfsprogs after I already did all these things, but I can incorporate your changes.